MultiLangProgGuide/MultiLangProgGuide.rst

```diff
@@ -1819,6 +1821,920 @@ If the requested :code:`timeout_nsec` exceeds
+  code.
+
+:code:`NodePrevS()` may also report another `error return code`_
+prefixed with :code:`C.`.
+
+Go SetValS()
+------------
+
+.. code-block:: go
+
+    SetValS(val *BufferT) int
+
+At the referenced local or global variable node, or the intrinsic
+special variable, :code:`SetValS()` wraps :code:`ydb_set_s()` to set
+the value specified by :code:`val`, with a return code of
+:code:`C.YDB_OK` or another `error return code`_ prefixed with
+:code:`C.`.
+
+Go SubNextS()
+-------------
+
+.. code-block:: go
+
+    SubNextS(sub *BufferT) int
+
+:code:`SubNextS()` wraps `ydb_subscript_next_s()` to facilitate
+breadth-first traversal of a local or global variable sub-tree.
+
+- If there is a node (either a node with data, or a subtree with or
+  without a node) in the subtree with a subscript following that of
+  the location specified by the :code:`KeyT` structure (neither a node
+  nor a subtree needs to exist at the location specified by the
+  :code:`KeyT` structure), :code:`SubNextS()` has a return code of
+  :code:`C.YDB_OK` and :code:`sub` references the value of that
+  subscript.
+- If there is no next node or subtree at that level of the subtree,
+  the method has a return code of :code:`C.YDB_OK` and
+  :code:`sub.GetLenUsed()` returns zero (i.e., the empty string).
+
+If there is an error, :code:`SubNextS()` has an `error return code`_
+prefixed with :code:`C.`.
+
+Go SubPrevS()
+-------------
+
+.. code-block:: go
+
+    SubPrevS(sub *BufferT) int
+
+:code:`SubPrevS()` wraps `ydb_subscript_previous_s()` to facilitate
+breadth-first traversal of a local or global variable sub-tree in
+reverse order.
+
+- If there is a node (either a node with data, or a subtree with or
+  without a node) in the subtree with a subscript preceding that of
+  the location specified by the :code:`KeyT` structure (neither a node
+  nor a subtree needs to exist at the location specified by the
+  :code:`KeyT` structure), :code:`SubNextS()` has a return code of
+  :code:`C.YDB_OK` and :code:`sub` references the value of that
+  subscript.
+- If the last subscript of the location specified by the :code:`KeyT`
+  structure is the empty string, :code:`sub` references the subscript
+  value of the last node of the subtree.
+- If there is no previous node at that level of the subtree, the method
+  has a return code of :code:`C.YDB_OK` and :code:`sub.GetLenUsed()`
+  returns zero (i.e., the empty string).
+
+Note that in the event an application uses empty string subscripts,
+and :code:`sub.GetLenUsed()` is zero, the application may need to
+invoke :code:`DataS()` to determine whether a node or subtree exists
+at that location.
+
+If there is an error, :code:`SubNextS()` has an `error return code`_
+prefixed with :code:`C.`.
+
+-----------------------
+Go Simple API Functions
+-----------------------
+
+Go LockS()
+----------
+
+.. code-block:: go
+
+    LockS( timeoutNsec uint64, ... *KeyT) int
```

Comment:
Space before timeoutNsec. Also should be parm name prior to '...' like "locklist" or some such.

Might want to note maximum here of ((#locks * 3) + 1) must be <= 36.

Reply:
Fixed. Also noted that the number of locks ≤11 (easier to parse than the formula).